content-src/components/DiscoveryStreamComponents/DSCard/DSCard.jsx

```diff
@@ -95,6 +95,7 @@ export class DSCard extends React.PureComponent {
           event: "CLICK",
           source: this.props.type.toUpperCase(),
           action_position: this.props.pos,
+          value: { card_type: this.props.flightId ? "spoc" : "organic" },
```

Comment:
Is value new?

Reply:
No, `value` is not new to this ping. The new part is the `card_type` for Top Stories events.

Reply:
So previously we were not sending a value in this ping but it's optional?

(I'm just confused because it looks like you're adding it here)

Reply:
Correct, it's an optional field, we didn't send it for Top Stories `CLICK` ping, but we did for other `CLICK` pings in Top Sites and Highlights. So yeah, in a sense, it's a new field to this particular ping.